app/konnect/org-management/audit-logging/reference.md

```diff
@@ -57,23 +78,29 @@ trace_id=8809518331550410226
 user_agent=grpc-node/1.24.11 grpc-c/8.0.0 (linux; chttp2; ganges)
 ```
 
-Each authorization log entry contains the following:
+Each authorization log entry contains the following attributes in addition to the defaults above:
 
 Property | Description
 ---------|-------------
-Timestamp | Time and date of the event in UTC.
-`service_name` | The name of the component within {{site.konnect_short_name}}. For example, `portals` or `runtimegroups`. In the above example, this appears as `Authz.portals`.
-`rt` | Milliseconds since Unix epoch.
-`src` | The IP address of the request originator.
 `action` | The type of action the user performed on the resource. For example, `retrieve`, `list`, or `edit`.
 `granted` | Boolean indicating whether the authorization was granted or not.
-`org_id` | The originating organization ID.
-`principal_id` | The user ID of the user that performed the action.
-`actor_id` | If using impersonation, the ID of the person doing the impersonation. Otherwise, this field is empty.
-`trace_id` | The trace ID of the request.
-`user_agent` | The user agent of the request: application, operating system, vendor, and version.
+
+
+## Access logs
+
+Access logs include information about requests to the Konnect API.
+
+Each access log entry contains the following attributes in addition to the defaults above:
+
+Property | Description
+---------|-------------
+`request` | The endpoint that was called.
+`query` | The request query parameters, if any.
+`act` | The HTTP request method; for example, `GET` or `POST`.
```

Comment:
note: We use `action` in authz logs and `act` in access logs. It might be nice to make them consistent. `action` is the clearer of the two.

Reply:
(note to self, no action on the docs front unless this changes)